web/src/components/filings-calendar/FilingsCalendarTaxAccess.test.tsx

```diff
@@ -202,6 +241,18 @@ describe("<FilingsCalendarTaxAccess />", () => {
     });
     await screen.findByTestId("tax-success");
     expect(screen.getByText(Config.taxCalendar.snackbarSuccessHeader)).toBeInTheDocument();
+    expect(screen.getByTestId("close-icon-button")).toBeInTheDocument();
+    jest.useFakeTimers();
+    act(() => {
+      jest.advanceTimersByTime(6000);
+    });
```

Comment:
I'm not sure this is necessary to test.

"Doesn't auto close alert" is a little strange to test because it only makes sense in the context of how it USED to work.

Someone new coming into the code wouldn't have that context, and this test wouldn't make any sense. It's drawing on assumptions the code USED to make but doesn't anymore. They'd be like "Why 6000 seconds?" - Because it USED to work that way, but doesn't anymore. 

At a certain point in the future, this would make as much sense as testing it "does not display 100 digits of pi" or whatever random thing. You'd just be like, sure, it doesn't, but why are we testing that?

It would be reasonable here to test that the close button works. No reason to test that the alert is NOT timed